POI.DiscordDotNet/Services/SlashCommandsManagementService.cs

```diff
@@ -32,6 +33,8 @@ public void Setup()
 			// TODO: Register slash commands below
 			_slashCommands.RegisterCommands<PingCommand>();
 			_slashCommands.RegisterCommands<UptimeCommand>();
+
```

Comment:
?

Reply:
Intentional, Ping and Uptime are both part of the Utils module while Birthday commands are part of the profile commands module ^^